Surface_mesh/include/CGAL/Surface_mesh/IO/PLY.h

```diff
@@ -305,7 +305,7 @@ class Surface_mesh_filler
 
     if (m_vcolors == 3)
     {
-      unsigned char r, g, b;
+      unsigned char r = 0, g = 0, b = 0;
```

Comment:
I think the overloads of `PLY_element::assign` should all end by `t = {}` to fill `t` with a default-constructed value, in case the property is missing. That would also fix that sort of warning.

Reply:
That's a good idea, I changed my fix by that.